content/en/platform/corda/4.4/open-source/node-operations-upgrade-cordapps.md

```diff
@@ -68,7 +68,7 @@ time.
 In order to perform the upgrade, follow the following steps:
 
 
-* If required, do a flow drain to avoid the definition of states or contracts changing whilst a flow is in progress (see [Flow drains](upgrading-cordapps.md#upgrading-cordapps-flow-drains) for more information)
+* If required, do a flow drain to avoid the definition of states or contracts changing whilst a flow is in progress (see [Flow drains](upgrading-cordapps.html#upgrading-cordapps-flow-drains) for more information)
```

Comment:
```suggestion
* If required, do a flow drain to avoid the definition of states or contracts changing whilst a flow is in progress (see [Flow drains](upgrading-cordapps.html#flow-drains) for more information)
```